.github/secrel/config.yml

```diff
@@ -10,21 +10,44 @@
 #   - -Dexample-build-arg=some-value
 #   - -Dexample.maven=${{ secrets.example_secret }}
 
+snyk:
+  additional-args:
+  - -PGITHUB_ACCESS_TOKEN=${{ secrets.ACCESS_TOKEN }}
```

Comment:
From https://github.com/department-of-veterans-affairs/lighthouse-di-demo-java-starterkit/commit/d48fecf8a622c3a397f37ddaf9416237efc17f28#diff-fd8faa09e718dcbb6de1bf4534d2c8aaac1c439366bbeeaaea6aef60e4dd474a